### ISSUE TITLE
WARNING — Antivirus-service may be unwantedly disabled — Update .env file on existing installations now!

### ISSUE THREAD

**Issue by Nebukadneza** [Mailu/Mailu#1597]
On 22.08.2020, we discovered (#1595) that a bug in the `mailu.env` file generated by https://setup.mailu.io/ did not enable the anti-virus service, even if the users did enable it. The `antivirus` container is started regardless of this, but not used, which is prone to give users a false feeling of security.

This holds true for `1.6`, `1.7` and current `master` installations which use `mailu.env` files generated through https://setup.mailu.io/

**For existing installations**, please edit your `mailu.env` file and change
```
# Antivirus solution (value: clamav, none)
#ANTIVIRUS=clamav
```

to

```
# Antivirus solution (value: clamav, none)
ANTIVIRUS=clamav
```

by removing the `#` from the beginning of the line. Then, restart your setup completely using `docker-compose up -d -V --force-recreate` or an alternative you know works for your setup.

**For new installations** (which plan to use `antivirus`), please refrain from using https://setup.mailu.io/ until #1596 is merged and backported, and `setup` is updated. We will give an update in this issue when this is done.
If you absolutely need to make a new installation, please follow the steps outlined above under _For existing installations_ immediately after installing.

Special thanks goes to @mirchaemanuel for the initially reporting and @Diman0 for swiftly diagnosing and fixing this issue!

Thank you for your patience until we fix this, and best regards,
- Your Mailu-Team

**Comment by Nebukadneza** [comment]
Update:

**For new installations**, https://setup.mailu.io/ has now been **fixed** and includes the updates files for `master` and `1.7`. You may now safely generate your setup-files, and benefit from a correctly enabled antivirus.

Thank you for your patience,and best regards,
- Your Mailu-Team

**Comment by Diman0** [comment]
Closing this issue since it is fixed for master (1.8) and 1.7 by PRs #1596 #1598
